### PR TITLE
feat(suite-native): show popup for view only enabling

### LINKED ISSUE #11886: View only - popup on device connect
When a user connects a Trezor and has never clicked "Skip", show modal that allows to turn on view-only mode right away.

It should be shown after "biometrics enable" modal (after onboarding) or after Connecting screen on subsequent runs of the app.

If user taps enable - we offer it again with new devices and passphrase wallets. 

If user taps skip  - we globally stop offering this feature. Save this in form of timestamp for possible timeout of this decision

image

[Figma](https://www.figma.com/file/KvbZGG16li6BA90Xtpf0BP/Device-interaction?type=design&node-id=970-17549&mode=design&t=KKhTAXTh0Wmu2KU9-4)

### PR DESCRIPTION
**Pull request by vytick**
Added popup to offer View only mode enabling. Do not show it for devices that do not have it enabled or when skip was clicked. 

Timestamp is used, so we can later measure time since skip was done and offer it again, after period of time if we want to. This is being considered as a future improvement.

## Related Issue

Resolve [#11886](https://github.com/trezor/trezor-suite/issues/11886)

https://github.com/trezor/trezor-suite/assets/2011829/9de52cff-2cf3-42d8-b471-4ad23f4ec6e0

### CODE REVIEW

File: suite-native/module-settings/src/slice.ts [hunk 1]
```diff
@@ -41,6 +44,9 @@ export const appSettingsSlice = createSlice({
         setBitcoinUnits: (state, { payload }: PayloadAction<PROTO.AmountUnit>) => {
             state.bitcoinUnits = payload;
         },
+        setRememberModeOfferChoiceTimestamp: state => {
+            state.rememberModeOfferChoiceTimestamp = new Date().getTime();
```
Comment (matejkriz) [inline suite-native/module-settings/src/slice.ts]: Keep reducer functions pure please, you can use `prepare` or pass it from the component...

File: suite-native/module-home/src/assets/DisconnectedTrezorDarkSvg.tsx
Comment (matejkriz) [inline suite-native/module-home/src/assets/DisconnectedTrezorDarkSvg.tsx:1]: nitpick: file name doesn't follow naming structure of other files in that folder where "dark" is given at the beginning e.g. DarkConnectedTrezorSvg.tsx, darkDashboard.png

File: suite-native/module-home/src/screens/HomeScreen/components/RememberDeviceBottomSheet.tsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+    const handleSetRememberModeOfferChoiceTimestamp = () =>
+        dispatch(setRememberModeOfferChoiceTimestamp());
+
+    const handleClose = () => {
+        handleSetRememberModeOfferChoiceTimestamp();
+    };
+
+    const handleEnable = () => {
+        handleSetRememberModeOfferChoiceTimestamp();
```
Comment (matejkriz) [inline suite-native/module-home/src/screens/HomeScreen/components/RememberDeviceBottomSheet.tsx]: I would not expect setting timestamp on enable and probably not even on close (only on clicking on skip) and I believe it is described like that in the issue description.

**Review by matejkriz — CHANGES_REQUESTED** [review]

File: suite-native/module-home/src/screens/HomeScreen/components/RememberDeviceBottomSheet.tsx (hunk 2 above)
Comment (vytick) [reply suite-native/module-home/src/screens/HomeScreen/components/RememberDeviceBottomSheet.tsx]: 73ad19a3b8

File: suite-native/module-home/src/assets/DisconnectedTrezorDarkSvg.tsx
Comment (vytick) [reply suite-native/module-home/src/assets/DisconnectedTrezorDarkSvg.tsx:1]: 73ad19a3b8

File: suite-native/module-settings/src/slice.ts (hunk 1 above)
Comment (vytick) [reply suite-native/module-settings/src/slice.ts]: 73ad19a3b8

File: suite-native/module-home/src/assets/DisconnectedTrezorDarkSvg.tsx
Comment (matejkriz) [reply suite-native/module-home/src/assets/DisconnectedTrezorDarkSvg.tsx:1]: nice solution 👍

**Comment by vytick** [comment]
@matejkriz Updated naming `remember` -> `viewOnly`

File: suite-native/module-home/src/screens/HomeScreen/components/EnableViewOnlyBottomSheet.tsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    const [isVisible, setIsVisible] = useState(false);
+
+    // show the bottom sheet if:
+    //     does not have view only mode enabled already
+    //     the device is authorized
+    //     not a portfolio tracker
+    //     the user hasn't made a choice yet
+    //     and biometrics initial setup was decided
+
+    const canBeShowed =
```
Comment (matejkriz) [inline suite-native/module-home/src/screens/HomeScreen/components/EnableViewOnlyBottomSheet.tsx:56]: feature flag has to be check here, right?

**Review by matejkriz — CHANGES_REQUESTED** [review]

File: suite-native/module-home/src/screens/HomeScreen/components/EnableViewOnlyBottomSheet.tsx (hunk 3 above)
Comment (vytick) [reply suite-native/module-home/src/screens/HomeScreen/components/EnableViewOnlyBottomSheet.tsx:56]: 1dadf9384b

**Review by matejkriz — APPROVED** [review]

**Pull request merged**